GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
this was a good catch.  somehow i missed the lack of javadoc earlier.  also the suggestion about adding an exception if called before setScript() is a good one.